FAQ.md

```diff
@@ -14,7 +14,7 @@
 
 ## Where does the name come from?
 
-The name *Slint* is derived from our design goals: **S**traightforward, **Li**ghtweight **N**ative **T**oolkit.
+The name *Slint* is derived from our design goals: **S**calable, **L**ightweight **I**ntuitive **N**ative **T**oolkit.
```

Comment:
Do we need to sprinkle some more commas over this sentence?

Reply:
changed to **S**calable, **L**ightweight, **I**ntuitive, and **N**ative **T**oolkit